### PR TITLE
Adding Allegro source and organization

### PR DESCRIPTION
**Pull request by przemoo83**

### CODE REVIEW

**Comment by przemoo83** [comment]
@googlebot I signed it!

File: organizations/allegro.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "allegro",
```
Comment (pfrisella) [inline organizations/allegro.json]: should be uppercase: ALLEGRO

File: sources/allegro.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "allegro",
+  "name": "Allegro",
+  "categories": ["ECOMMERCE", "SALES"],
+  "organization": "Allegro",
```
Comment (pfrisella) [inline sources/allegro.json]: organization ID should be uppercase: ALLEGRO

File: sources/allegro.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "allegro",
```
Comment (pfrisella) [inline sources/allegro.json]: ALLEGRO

File: sources/allegro.json (hunk 2 above)
Comment (pfrisella) [reply sources/allegro.json]: source Id was fixed but you missed the organization id, it should match and also be uppercase ALLEGRO

"organization": "Allegro", -> "organization": "ALLEGRO",

**Pull request merged**